### DISCUSSION TITLE
Auxiliary Timer view

### DISCUSSION THREAD

**Discussion by MaikWaschfeld** [cpvalente/ontime#1122] [Q&A] [answered]
Hi ontime team,

I appreciate your efforts for such an amazing software and like to digg into it!

For my "Auxiliary Timer" question:
is there a way to get the 'Auxiliary timer's readout to a 'view'?
I couldn’t really find any concrete handlers in the docs, like for the ’Stage Timer’ view.

I tried <MY-IP-ADDRESS:4001/> and:
'… /aux’, '… /aux1’, '… /aux/‘, '… /aux/1’, '… /aux/1/',  '… /auxtimer', '… /auxtimer1', '… /auxtimer/‘, '… /auxtimer/1’, '… /auxtimer/1/‘, but I always get „redirected" to '… /timer’.

Where's my error?
Any help is appreciated!

Regards ...
... Maik

**Comment by cpvalente** [comment, marked as answer]
Hi @MaikWaschfeld , thank you for this question

As of now, it is just something that the editor can use to run the show and is only visible in the editor view
We have plans to extend this but the work will not start until after summer

As for the URLs, we dont keep any secret URLs, all of the views are in the menu (asides from the custom view demo which is at `/external/demo` https://docs.getontime.no/features/custom-views/)

**Reply by MaikWaschfeld** [reply]
Hi @cpvalente. thank you for this information!

It's ready, when it's ready and that's fine.
